### PR TITLE
Asking for review on the Final version of the project "The-New-York-Times-Article"

### PR DESCRIPTION
**Pull request by Maha-Magdy**
what I did in this project to review is:
1. create the whole skeleton using HTML.
2. style the article using CSS, images, and adding some fonts.

### CODE REVIEW

**Review by SmithVyne — CHANGES_REQUESTED** [review]
INVALID PULL REQUEST  ❌ 
Hello, you need to make this pull request more professional
[REQUIRED] -  Make your pull request title and the body more meaningful. Preferably, use the title of this project and highlight what you did. [Screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png)
[REQUIRED] - Also, your linters are failing. Try to fix them
[REQUIRED] - Use correct git-flow. `main` , `development`, and `feature banches`

**Review by christianotieno — CHANGES_REQUESTED** [review]
## Changes Required ♻️ 
Hello @Maha-Magdy 👋 

The project looks great. There are some missing sections that need implementation before moving forward. I have highlighted them below. 👇 

### Happy coding 💻 
#### Christian Otieno (CR)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,782 @@
...
+                <li><a class="swap-text-decoration small-text franklin-font" href="#">T Brand Studio</a></li>
+                <li><a class="swap-text-decoration small-text franklin-font" href="#">Your Ad Choices</a></li>
+                <li><a class="swap-text-decoration small-text franklin-font" href="#">Privacy Policy</a></li>
+                <li><a class="swap-text-decoration small-text franklin-font" href="#">Terms of Service</a></li>
+                <li><a class="swap-text-decoration small-text franklin-font" href="#">Terms of Sale</a></li>
+                <li><a class="swap-text-decoration small-text franklin-font" href="#">Site Map</a></li>
+                <li><a class="swap-text-decoration small-text franklin-font" href="#">Help</a></li>
+                <li><a class="swap-text-decoration small-text franklin-font" href="#">Subscriptions</a></li>
+            </ul>
+        </nav>
```
Comment (christianotieno) [inline index.html:1027]: ## Implement missing sections
Implement the following missing sections 👇 
- [ ] `Suggested newsletters for you` Section.
- [ ] `Most Popular` Section.
- [ ] `More in Space and Astronomy` Section.
- [ ] `Editors’ Picks` Section - That is beneath the `More in Space and Astronomy` Section.
- [ ] The sticky bottom footer just as it appears in the [Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html). 
- footer

**Review by christianotieno — CHANGES_REQUESTED** [review]
#### NOTE: 
Before making the changes. I would suggest you open a new `feature`  branch from the `development` branch and work from there. Your next review should be from this newly created `feature`  branch ready to be merged to this `development` branch upon approval. This is so as to adhere to the recommended `git-flow` workflow. You should not be working on the `development `branch at all time.

**Pull request closed without merging**